jetty-nosql/src/main/java/org/eclipse/jetty/nosql/mongodb/MongoSessionIdManager.java

```diff
@@ -179,55 +184,60 @@ public MongoSessionIdManager(Server server, DBCollection sessions)
                 BasicDBObjectBuilder.start().add("id",1).get(),
                 BasicDBObjectBuilder.start().add("unique",true).add("sparse",false).get());
         _sessions.ensureIndex(
-                BasicDBObjectBuilder.start().add("id",1).add("version",1).get(),
-                BasicDBObjectBuilder.start().add("unique",true).add("sparse",false).get());
+                BasicDBObjectBuilder.start().add("id", 1).add("version", 1).get(),
```

Comment:
There's quite a few places in the patch where it seems that just whitespaces changes have been made, making it a little difficult to see if any actual code change has been made.